web_responsive/static/src/css/web_responsive.scss

```diff
@@ -576,3 +578,83 @@ html .o_web_client .o_main .o_main_content {
 .oe_wait {
   cursor: progress;
 }
+
+// Document Viewer
+.o_web_client.o_chatter_position_sided {
+    .o_document_viewer {
+        z-index: 10 !important;
```

Comment:
Can you avoid somehow the `!important` part for not forcing so many styles?

Reply:
Usually the way to do this is to use a more specific selector, or use the same selector but later.

BTW please add some comments to understand what do these things do.